src/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -1687,30 +1689,17 @@ public boolean hasSymbolicAlleles() {
     }
 
     public static boolean hasSymbolicAlleles( final List<Allele> alleles ) {
-        for ( final Allele a: alleles ) {
-            if (a.isSymbolic()) {
-                return true;
-            }
-        }
-        return false;
+        return alleles.stream().anyMatch(a->a.isSymbolic());
     }
 
     public Allele getAltAlleleWithHighestAlleleCount() {
-        // optimization: for bi-allelic sites, just return the 1only alt allele
+        // optimization: for bi-allelic sites, just return the only alt allele
         if ( isBiallelic() )
             return getAlternateAllele(0);
 
-        Allele best = null;
-        int maxAC1 = 0;
-        for ( Allele a : getAlternateAlleles() ) {
-            final int ac = getCalledChrCount(a);
-            if ( ac >= maxAC1 ) {
-                maxAC1 = ac;
-                best = a;
-            }
-
-        }
-        return best;
+        return getAlternateAlleles().stream()
+                .max((a1, a2) -> Integer.compare(getCalledChrCount(a1), getCalledChrCount(a2)))
```

Comment:
While this is neater, it's going to be a lot slower on big VCFs with many samples and multiple alleles, since `getCalledChrCount(allele)` traverses all the genotypes each time it's called.